### PR TITLE
Fix for a bug where services of networks weren't being fetched for world news

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/FindByPid/TlecController.php [hunk 1]
```diff
@@ -86,7 +86,12 @@ public function __invoke(
             $galleries = $aggregationService->findDescendantGalleries($programme, 4);
         }
 
-        $lastOn = $collapsedBroadcastsService->findPastByProgramme($programme, 1);
+        if ($programme->getNetwork() && $programme->getNetwork()->isWorldNews()) {
```
Comment (votemike) [inline src/Controller/FindByPid/TlecController.php]: Are you going to use the new `isWorldNews()` method in the MapPresenter?

**Comment by votemike** [comment]
Maybe a test that fails without this change?

File: src/Controller/FindByPid/TlecController.php [hunk 2]
```diff
@@ -181,6 +171,21 @@ private function getComingSoonPromotion(ImagesService $imagesService, ProgrammeC
         );
     }
 
+    private function getLastOn(ProgrammeContainer $programme, CollapsedBroadcastsService $collapsedBroadcastsService)
```
Comment (votemike) [inline src/Controller/FindByPid/TlecController.php]: return Typehint?

**Pull request merged**